pkgs/applications/audio/gmpc/default.nix

```diff
@@ -56,6 +67,17 @@ stdenv.mkDerivation rec {
     hicolor-icon-theme
   ];
 
+  desktopItems = [
+    (makeDesktopItem {
```

Comment:
But sources contains data/gmpc.desktop.in, can we just install it?

Reply:
Ah, you are right. After looking into this I found the root cause, why the gmpc.desktop seemed to be missing in my desktop environment. Was an unrelated, local issue/ nix beginner mistake: Errors in XDG_DATA_DIRS environment variable. Closing this PR